### PR TITLE
reopening PR for adding  counties / states with CARTO sql api

### PR DESCRIPTION
**Pull request by timhitchins**

### CODE REVIEW

**Comment by timhitchins** [comment]
@clhenrick Here is the new PR to test.  I _think_ I did this right.  This would revert #33.

cc @azadag @arghgr @goldfarb @erinmcel

File: script.js [hunk 1]
```diff
@@ -23,8 +23,17 @@ const renStikeSheetId = "1rCZfNXO3gbl5H3cKhGXKIv3samJ1KC4nLhCwwZqrHvU";
 const moratoriumSheetURI = `https://docs.google.com/spreadsheets/d/${moratoriumSheetId}/export?format=csv&id=${moratoriumSheetId}`;
 const rentStrikeSheetURI = `https://docs.google.com/spreadsheets/d/${renStikeSheetId}/export?format=csv&id=${renStikeSheetId}`;
 
-// states geojson url
-const statesGeoJsonURI = "./states.geojson";
+// table in CARTO that syncs with the Google sheet data
+// note the sync is broken due to a problem with sheets
+// that reference data from another sheet
+const cartoSheetSyncTable =
+  "emergency_tenant_protections_carto_sync_do_not_edit";  // this table is not syncing and will need to chage potentially
```
Comment (azadag) [inline script.js:30]: change to "emergency_tenant_protections_current_do_not_edit_me_sheet1"

**Comment by timhitchins** [comment]
Pretty sure I did this right if I was understanding things correctly.  Did a `git cherry-pick` of 95e9bde and also updated the CARTO sync table the new one @azadag made.  However we are still missing US territories (Puerto Rico).  

cc @clhenrick

**Comment by timhitchins** [comment]
Ok.  Puerto Rico is in there now.

**Comment by clhenrick** [comment]
Thanks @timhitchins this looks good. The commit I meant to ask you to cherry-pick actually was 4859084 which I just added. It looks like there's an odd merge conflict happening though with master. I'm not sure why, but it's messing up the `script.js` file. It may be because something got messed up with git's history.

**Comment by clhenrick** [comment]
@timhitchins okay so I think I fixed the merge conflict, I think this is good to merge but we should double check that the new table sync is still working (cc @azadag) and that the research team is aware of this change, mainly that their edits will no longer immediately show up.

**Comment by clhenrick** [comment]
@timhitchins @azadag I'm closing this one in favor of #36 as it keeps the git history a little cleaner.

**Pull request closed without merging**